go.mod

```diff
@@ -29,6 +29,6 @@ replace (
 	// Required to replace version with vulnerabilities.
 	github.com/coreos/etcd v3.3.10+incompatible => github.com/coreos/etcd v3.3.24+incompatible
 	github.com/gorilla/websocket v1.4.0 => github.com/gorilla/websocket v1.4.2
-	sigs.k8s.io/cluster-api v0.3.9 => github.com/giantswarm/cluster-api v0.3.9-gs
+	sigs.k8s.io/cluster-api v0.3.10 => github.com/giantswarm/cluster-api v0.3.10-gs
```

Comment:
you can remove the version number from the left side, so the version is only written in one place